flujo/validation/linters.py

```
@@ -0,0 +1,1522 @@
+    - V-I6: inherit conversation consistency
+    """
+
+    _ALLOWED_PARENT_ROOTS = {
+        "scratchpad",
+        "command_log",
+        "hitl_history",
+        "conversation_history",
+        "yaml_text",
+        "generated_yaml",
+        "run_id",
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)